### PR TITLE
added new variables for raw materials demand

### PR DESCRIPTION
**Pull request by fbenke-pik**

### CODE REVIEW

File: definitions/variable/variables.yaml [hunk 1]
```diff
@@ -10550,3 +10550,423 @@
...
+- Minerals|Aluminium|Power Plants (renewable):
+    description: Demand of Aluminium for Power Plants (renewable)
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (fossil + nuclear):
+    description: Demand of Aluminium for Power Plants (fossil + nuclear)
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Batteries:
+    description: Demand of Aluminium for Stationary Batteries
+    unit: Mt/yr
+- Minerals|Aluminium|Electrolyers:
```
Comment (danielhuppmann) [inline definitions/variable/variables.yaml:10574]: I guess there's a typo here? (and also in the other variables for this electrolyser demand)
```suggestion
- Minerals|Aluminium|Electrolysers:
```

File: definitions/variable/variables.yaml [hunk 2]
```diff
@@ -10550,3 +10550,423 @@
...
+- Minerals|Aluminium|CSP:
+    description: Demand of Aluminium for CSP Plants
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Fuel Cells:
+    description: Demand of Aluminium for Stationary Fuel Cells
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (renewable):
+    description: Demand of Aluminium for Power Plants (renewable)
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (fossil + nuclear):
```
Comment (danielhuppmann) [inline definitions/variable/variables.yaml:10568]: Maybe call this "Thermal Power Plants"?

File: definitions/variable/variables.yaml [hunk 3]
```diff
@@ -10550,3 +10550,423 @@
...
+- Minerals|Aluminium|PV:
+    description: Demand of Aluminium for PV Power Plants
+    unit: Mt/yr
+- Minerals|Aluminium|CSP:
+    description: Demand of Aluminium for CSP Plants
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Fuel Cells:
+    description: Demand of Aluminium for Stationary Fuel Cells
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (renewable):
```
Comment (danielhuppmann) [inline definitions/variable/variables.yaml:10565]: If it's a thermal power plant, can you really distinguish the plants between fossil gas vs. biogas?

**Review by danielhuppmann — COMMENTED** [review]
I think there's a typo here

**Comment by danielhuppmann** [comment]
I assume you'll leave the power-plants sub-category variables as is?

**Comment by fbenke-pik** [comment]
> I assume you'll leave the power-plants sub-category variables as is?

I asked Frederike to comment on your open questions, so let's wait for her response.

**Comment by frederikebartels** [comment]
> I assume you'll leave the power-plants sub-category variables as is?

I would leave it as it is, because I already aligned it with the colleague who will report these results. Hopefully, he carefully thought about it...

**Review by danielhuppmann — APPROVED** [review]
Ok, thanks!

**Pull request merged**